notebooks/samples/explanations/ai-explanations-image-tf2.ipynb

```diff
@@ -0,0 +1,1758 @@
+{
```

Comment:
<h3>Send the explanations request</h3><p> </p><p>If you deployed both an IG and an XRAI model, you can request predictions for both models and compare the results. </p><p> </p><p><strong>If you only deployed one model above, run only the cell for that explanation method.</strong></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/192/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='74'/>